### ISSUE TITLE
chrome browser always on the top when toggle zoom-fullscreen of other window

### ISSUE THREAD

**Issue by realfirst** [koekeishiya/yabai#1243]
chrome browser always on the top when toggle zoom-fullscreen of other window

this is a gif screen record of this phenomeno

https://photos.app.goo.gl/UJ1xEL7FMcSiCHMx7

**Comment by realfirst** [comment]
yabai-v4.0.0
macOS Monterey version 12.3(21E230)

**Comment by dumbbyte** [comment]
For what it is worth, I'm experiencing this same behavior with Safari.

yabai-v4.0.0
macOS 12.3.1 (21E258)

Happy to help troubleshoot this, I'm just not really sure where to begin.

**Comment by armand1m** [comment]
I'm also having this same issue. It's pretty annoying, usually the only thing that helps me fix this is closing chrome and reopening it. This seems to also affect popups from chrome (SSL popup, camera/microphone permissions, etc), where they keep behind the main chrome window so you can never actually see them 

yabai-v4.0.0
macOS 12.3.1
Apple M1 Pro

**Comment by zepster** [comment]
I'm also having this same issue with chrome


yabai-v4.0.0
macOS 12.3.1

**Comment by realfirst** [comment]
> disable topmost: `yabai -m config window_topmost off`

Hi, [koekeishiya](https://github.com/koekeishiya)
i change yabai config to disable topmost but phenomenon still exists
<img width="349" alt="image" src="https://user-images.githubusercontent.com/174605/169626882-dccd0f50-709b-4d47-afdc-b568b3b4b152.png">

**Comment by prgres** [comment]
Have the same problem when using stacK. Sometimes chrome "freeze" and only way is to close that window which is pretty annoying.

**Comment by nzhl** [comment]
Any progress on this

**Comment by Try431** [comment]
I can confirm this is still happening for me. Chrome is always on top, for both `--toggle zoom-parent` and `--toggle zoom-fullscreen`.

```
% yabai --version
yabai-v5.0.1
% sw_vers
ProductName:	macOS
ProductVersion:	12.6
BuildVersion:	21G115
```

**Comment by koekeishiya** [comment]
Fixed in v6.0.0
